src/components/button-toggle/button-toggle.component.tsx

```diff
@@ -28,29 +49,27 @@ const ButtonToggle = (props) => {
     onBlur,
     onChange,
     onFocus,
-    size,
+    size = "medium",
     value,
   } = props;
   const { onMouseEnter, onMouseLeave } = useContext(InputGroupContext);
 
   const inputGuid = guid();
-  const inputRef = useRef(null);
+  const inputRef = useRef<HTMLInputElement>(null);
   let icon;
 
   if (buttonIcon) {
     icon = (
       <ButtonToggleIcon
-        aria-hidden
         buttonIcon={buttonIcon}
         buttonIconSize={buttonIconSize}
-        hasNoContent={!children}
```

Comment:
Why are both `aria-hidden` and `hasNoContent` props removed?

Reply:
`aria-hidden` could be applied directly to the `Icon` instead of passing it down from here (please check the `button-toggle-icon.component.tsx` file).  As for `hasNoContent` - this prop was redundant, all it was doing was that:
https://github.com/Sage/carbon/pull/5365/files#diff-9c787793b3c94e782a732088653da4f22cc3491425db3d5a70eb601ff1ce7d85L114-L120